renderer/Interaction.cpp

```diff
@@ -30,6 +31,60 @@ idInteraction implementation
 idCVar r_useInteractionTriCulling("r_useInteractionTriCulling", "1", CVAR_RENDERER | CVAR_BOOL, "1 = cull interactions tris");
 idCVarInt r_singleShadowEntity( "r_singleShadowEntity", "-1", CVAR_RENDERER, "suppress all but one shadowing entity" );
 
+void idInteraction::PrepareLightSurf( drawSurf_s **link, const srfTriangles_t *tri, const viewEntity_s *space,
```

Comment:
At the moment, it's still used in `R_AddLightSurfaces`, which has not been parallelized so far. We might do that in the future (it is the next biggest "offender" in the frontend, although with some distance to `R_AddModelSurfaces`), but that should probably be its own PR :)

So yeah, I guess I didn't want to touch `R_AddLightSurfaces` in this PR, hence why the function is still around.

Reply:
I think that function needs to be removed. If not here, then by the other change soon. It contains some logic which is dangerous to have in two places.

Reply:
Fair point, I've removed the code duplication, now only the linking part takes a different turn depending on the caller. (Which it has to, anyway, because the usage in R_AddLightSurfs is a bit different.)